lib/ethers/contract.ex

```diff
@@ -27,7 +27,10 @@ defmodule Ethers.Contract do
   data = MyProject.Contract.example_function(...)
 
   # Use data to handle eth_call
-  Ethers.Contract.call(data, to: "0xADDRESS", from: "0xADDRESS")
+  # Note that Ethers.call/2 requires a JSON_RPC_URL configured in your config.exs as below
+  # config :ethereumex, :url, "https://ethereum-sepolia.publicnode.com"
```

Comment:
I actually don't think this is the right place for this sentence. On top of that this actually is not exactly true as one can choose to always pass in the url in overrides argument of `Ethers.call/2`. I think it is better to remove this.